src/views/detail/detail.js

```diff
@@ -1,12 +1,13 @@
 import Vue from 'vue'
+import 'regenerator-runtime/runtime'
 
 import Sidebar from '../../components/Sidebar'
 import CodeSnippet from '../../components/CodeSnippet'
 
 import './detail.pug'
 import './detail.scss'
 
-import mock from '../../utils/mock'
```

Comment:
Can you delete this mock?

Reply:
Yes, now the component is not a mock, it's being called from Github 😄